pkg/api/mlflow/service/artifact/storage/local_test.go

```diff
@@ -7,8 +7,82 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/config"
 )
 
+// TestGetArtifact for the local storage implementation.
+func TestGetArtifact_Ok(t *testing.T) {
+	// setup
+	defaultArtifactRoot := t.TempDir()
+	runArtifactRoot := "/run-artifact-root/"
+	fileName := "file.txt"
+	fileContent := "artifact content"
+
+	err := os.MkdirAll(filepath.Join(defaultArtifactRoot, runArtifactRoot), os.ModePerm)
+	assert.Nil(t, err)
+
+	f, err := os.Create(filepath.Join(defaultArtifactRoot, runArtifactRoot, fileName))
+	assert.Nil(t, err)
+	_, err = f.Write([]byte(fileContent))
+	assert.Nil(t, err)
+
+	// invoke
+	config := &config.ServiceConfig{
+		DefaultArtifactRoot: defaultArtifactRoot,
+	}
+	storage, err := NewLocal(config)
```

Comment:
you can pass `nil` to `NewLocal` (or an empty config if you prefer) — the default artifact root is not used anymore by the storage providers since my fixes

Reply:
will do